Adding support for multiple git providers

ryanneufeld opened the pull request:
I'm sure this could be improved, but for now it works with github as it always has, and works in a limited capacity with bitbucket.  But enough to at least know which branch was effected if not the tag.

Commit:
Adding support for multiple git providers

ryanneufeld committed:
Fixing pull#3 coding standards violations.

ryanneufeld committed:
Further fixes for pull#3

ryanneufeld committed:
Pull#3 - Putting undefined vars on one line

ryanneufeld committed:
Testing osx keychain

ryanneufeld committed:
Testing osx keychain

scottgonzalez commented:
This module has undergone some major updates and I don't think Bitbucket support is viable anymore.

ryanneufeld commented:
I suggest renaming it to node-github-notifier then. As the name is
potentially misleading.
On Dec 22, 2015 10:49, "Scott González" <notifications@github.com> wrote:

> This module has undergone some major updates and I don't think Bitbucket
> support is viable anymore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scottgonzalez/node-git-notifier/pull/3#issuecomment-166701718>
> .
>

scottgonzalez commented:
Agreed. I'll rename and do a release so people can switch over to the new name.

scottgonzalez commented:
I've renamed the repo, published github-notifier 2.2.0, and deprecated all versions of git-notifier with a message that the module has been renamed. Here's the output of installing git-notifier now:

```
$ npm install git-notifier
npm WARN deprecated git-notifier@2.1.0: renamed to github-notifier
git-notifier@2.1.0 node_modules/git-notifier
└── eventemitter2@0.4.14
```

ryanneufeld commented:
Nice!


--Ryan Neufeld BSc CIS
ryan@neufeldmail.com

On Wed, Dec 23, 2015 at 9:28 AM, Scott González <notifications@github.com>
wrote:

> I've renamed the repo, published github-notifier 2.2.0, and deprecated all
> versions of git-notifier with a message that the module has been renamed.
> Here's the output of installing git-notifier now:
>
> $ npm install git-notifier
> npm WARN deprecated git-notifier@2.1.0: renamed to github-notifier
> git-notifier@2.1.0 node_modules/git-notifier
> └── eventemitter2@0.4.14
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scottgonzalez/node-github-notifier/pull/3#issuecomment-166950244>
> .
>
